crlite_query/__init__.py

```diff
@@ -412,29 +412,14 @@ def __str__(self):
     def is_revoked(self):
         return self.state == "Revoked"
 
-    def result_icon(self):
-        if self.state == "Revoked":
-            return "⛔️"
-        if self.state == "Not Enrolled":
-            return "❔"
-        if self.state == "Valid":
-            return "👍"
-        if self.state == "Too New":
-            return "🐇"
-        if self.state == "Expired":
-            return "⏰"
-        if self.state == "Unknown Issuer":
-            return "⁉️"
-        return ""
-
     def print_query_result(self, *, verbose=0):
         padded_name = self.name + " " * 5
         padding = "".ljust(len(padded_name))
 
         if not self.issuer:
             self.state = "Unknown Issuer"
         else:
-            enrolled_icon = "✅" if self.issuer["crlite_enrolled"] else "❌"
+            enrolled_icon = "yes" if self.issuer["crlite_enrolled"] else "no"
```

Comment:
😞
(probably for the best, though)
also, actually, `enrolled_icon` probably isn't the right name any longer

Reply:
```
       __               
  ____/ /___  ____  ___ 
 / __  / __ \/ __ \/ _ \
/ /_/ / /_/ / / / /  __/
\__,_/\____/_/ /_/\___/ 
                        
```